File: PocketForecast/UserInterface/Utils/UIFont+ApplicationFonts.m
////////////////////////////////////////////////////////////////////////////////
//
//  JASPER BLUES
//  Copyright 2013 Jasper Blues
//  All Rights Reserved.
//
//  NOTICE: Jasper Blues permits you to use, modify, and distribute this file
//  in accordance with the terms of the license agreement accompanying it.
//
////////////////////////////////////////////////////////////////////////////////




#import "UIFont+ApplicationFonts.h"


@implementation UIFont (ApplicationFonts)

+ (UIFont*)applicationFontOfSize:(CGFloat)size
{
    return [UIFont fontWithName:@"Varela Round" size:size];
}

+ (UIFont*)temperatureFontOfSize:(CGFloat)size
{
    return [UIFont fontWithName:@"Questrial" size:size];
}


@end

File: PocketForecast/UserInterface/Utils/UIFont+ApplicationFonts.swift
////////////////////////////////////////////////////////////////////////////////
//
//  TYPHOON FRAMEWORK
//  Copyright 2013, Typhoon Framework Contributors
//  All Rights Reserved.
//
//  NOTICE: The authors permit you to use, modify, and distribute this file
//  in accordance with the terms of the license agreement accompanying it.
//
////////////////////////////////////////////////////////////////////////////////

import Foundation

extension UIFont {
    
    public class func applicationFontOfSize(size : CGFloat) -> UIFont {
        return UIFont(name: "Varela Round", size: size)!
    }
    
    public class func temperatureFontOfSize(size : CGFloat) -> UIFont {
        return UIFont(name: "Questrial", size: size)!
    }
        
}
